packages/navigator/config/index.js

```diff
@@ -12,19 +12,29 @@ module.exports = {
     assetsPublicPath: '/',
     proxyTable: {
       '/login': {
-        target: 'http://localhost:8080'
+        target: 'https://master.dev.molgenis.org',
+        changeOrigin: true,
+        secure: false
       },
       '/api': {
-        target: 'http://localhost:8080'
+        target: 'https://master.dev.molgenis.org',
+        changeOrigin: true,
+        secure: false
       },
       '/menu': {
-        target: 'http://localhost:8080'
+        target: 'https://master.dev.molgenis.org',
+        changeOrigin: true,
+        secure: false
       },
       '/plugin': {
-        target: 'http://localhost:8080'
+        target: 'https://master.dev.molgenis.org',
+        changeOrigin: true,
+        secure: false
       },
       '/files': {
-        target: 'http://localhost:8080'
+        target: 'https://master.dev.molgenis.org',
+        changeOrigin: true,
+        secure: false
```

Comment:
You probably wanted to test the navigator against master.dev, but this change is unrelated to fixing the build? If we move the navigator in DE2, this whole file is redundant anyway right?

Reply:
this pr does not move the navigator into de DE2
testing agains the master.dev is the right default here imo as de navigator needs data to test/dev